pkg/meta/annotations.go

```diff
@@ -197,6 +197,15 @@ const (
 	NMAHealthProbeInitialDelaySeconds = "initial-delay-seconds"
 	//
 	// Use GenNMAHealthProbeAnnotationName to generate the name.
+
+	// Set of annotations for scrutinize
+	//
+	ScrutinizePodTimeToLive             = "vertica.com/scrutinize-pod-ttl"
+	ScrutinizePodTimeToLiveDefaultValue = 1800 // 30min
+	ScrutinizePodRestartPolicy          = "vertica.com/scrutinize-pod-restart-policy"
+	RestartPolicyAlways                 = "Always"
```

Comment:
Can we use the constants from corev1? `RestartPolicyAlways`, `RestartPolicyOnFailure` and `RestartPolicyNever`

Reply:
I used this because the annotation is a string, the caller will convert it into `corev1.RestartPolicy`.

Reply:
We should still be able to use the ones from `corev1` as part of the validation. You just need to cast them to strings.